doc/appendix-de-kassensichv/examples/pos-receipt.md

```diff
@@ -0,0 +1,468 @@
+
+
+implicit flow
+
+ReciptRequest
+
+`
+{
+  "ftCashBoxID": "cashboxid-guid",
+  "ftPosSystemId": "possystemid-guid",
+  "cbTerminalID": "terminalid",
+  "cbReceiptReference": "233348",
+  "cbReceiptMoment": "2019-10-25T13:32:45.133Z",
+  "cbChargeItems": [
+    {
+      "Position": 1,
+      "Quantity": 1.0,
+      "Description": "Bier 0,5 lt",
+      "Amount": 3.80,
+      "VATRate": 19.0000,
+      // 0x4445000000000011 (delivery normal)
+      "ftChargeItemCase": 4919338167972134929,
+      "ftChargeItemCaseData": "",
+      "VATAmount": 0.61,
+      "CostCenter": "1",
+      "AccountNumber": "Accountnumber",
+      "ProductGroup": "Bier",
+      "ProductNumber": "101",
+      "ProductBarcode": "",
+      "Unit": "Stk",
+      "UnitQuantity": 1,
+      "UnitPrice": 3.8,
+      "Moment": "2019-10-25T13:32:45.133Z"
+    },
+    {
+      "Position": 2,
+      "Quantity": 1.0,
+      "Description": "Schnitzel",
+      "Amount": 9.20,
+      "VATRate": 7.00,
+      // 0x4445000000000012 (delivery reduced-1)
+      "ftChargeItemCase": 4919338167972134930,
+      "ftChargeItemCaseData": "",
+      "VATAmount": 0.60,
+      "CostCenter": "1",
+      "AccountNumber": "Accountnumber",
+      "ProductGroup": "Speisen",
+      "ProductNumber": "102",
+      "ProductBarcode": "",
+      "Unit": "Stk",
+      "UnitQuantity": 1,
+      "UnitPrice": 9.20,
+      "Moment": "2019-10-25T13:32:45.133Z"
+    }
+  ],
+  "cbPayItems": [
+    {
+      "Position": 1,
+      "Quantity": 1,
+      "Description": "Bar",
+      "Amount": 13.0,
+      // 0x4445000000000001 (cash payment)
+      "ftPayItemCase": 4919338167972134913,
+      "ftPayItemCaseData": "",
+      "AccountNumber": "Accountnumber",
+      "CostCenter": "Costcenter123",
+      "MoneyGroup": "BAR",
+      "MoneyNumber": "1234",
+      "Moment": "2019-10-25T13:32:45.133Z"
+    }
+  ],
+  // 0x4445 0000 0000 0001 (pos-receipt) + 0x0000 0001 0000 0000 (implicit flow)
+  "ftReceiptCase": 4919338168240570369,
```

Comment:
I think this is wrong :) `4919338168240570369 `=> ‭`0x4445 0000 1000 0001`‬ when it should be `0x4445 0001 0000 0001` so I guess this should be `4919338172267102209‬`

Reply:
fixed